[Bug] GUI fails to start since commit 5fd8c29

**Describe the bug**
On the current master branch since commit 5fd8c29 the GUI fails to start with the following output:
```text
$ grass79 --gui
Starting GRASS GIS...
Cleaning up temporary files...

          __________  ___   __________    _______________
         / ____/ __ \/   | / ___/ ___/   / ____/  _/ ___/
        / / __/ /_/ / /| | \__ \\_  \   / / __ / / \__ \
       / /_/ / _, _/ ___ |___/ /__/ /  / /_/ // / ___/ /
       \____/_/ |_/_/  |_/____/____/   \____/___//____/

Welcome to GRASS GIS 7.9.dev (547ded6c0)
GRASS GIS homepage:                      https://grass.osgeo.org
This version running through:            Bash Shell (/bin/bash)
Help is available with the command:      g.manual -i
See the licence terms with:              g.version -c
See citation options with:               g.version -x
If required, restart the GUI with:       g.gui wxpython
When ready to quit enter:                exit

Launching <wxpython> GUI in the background, please wait...
To run a command as administrator (user "root"), use "sudo <command>".
See "man sudo_root" for details.

GRASS UK_ALOS_reprojected/PERMANENT:~ > Traceback (most recent call last):
  File "/home/denis/grass_exec/grass79/gui/wxpython/wxgui.py", line 106, in OnInit
    workspace=self.workspaceFile)
  File "/home/denis/grass_exec/grass79/gui/wxpython/lmgr/frame.py", line 151, in __init__
    self.notebook = self._createNoteBook()
  File "/home/denis/grass_exec/grass79/gui/wxpython/lmgr/frame.py", line 389, in _createNoteBook
    parent=self.notebook, giface=self._giface, simpleEditorHandler=self.OnSimpleEditor)
  File "/home/denis/grass_exec/grass79/gui/wxpython/lmgr/pyshell.py", line 52, in __init__
    useStockId=(sys.platform != "darwin"))
  File "/usr/lib/python3/dist-packages/wx/py/shell.py", line 268, in __init__
    *args, **kwds)
TypeError: __init__() got an unexpected keyword argument 'useStockId'
OnInit returned false, exiting...
Error in atexit._run_exitfuncs:
wx._core.wxAssertionError: C++ assertion "GetEventHandler() == this" failed at ../src/common/wincmn.cpp(478) in ~wxWindowBase(): any pushed event handlers must have been removed

GRASS UK_ALOS_reprojected/PERMANENT:~ > exit
Cleaning up default sqlite database ...
Cleaning up temporary files...
Done.

Goodbye from GRASS GIS
```

**System description (please complete the following information):**
- Operating System: GNU/Linux Ubuntu 18.04 64-bit
- GRASS GIS version: master branch (currently at commit 31113f1)

--- a/gui/wxpython/lmgr/pyshell.py
+++ b/gui/wxpython/lmgr/pyshell.py
@@ -45,17 +45,17 @@ class PyShellWindow(wx.Panel):
         self.intro = _("Welcome to wxGUI Interactive Python Shell %s") % VERSION + "\n\n" + \
             _("Type %s for more GRASS scripting related information.") % "\"help(gs)\"" + "\n" + \
             _("Type %s to add raster or vector to the layer tree.") % "\"AddLayer()\"" + "\n\n"
+
+        shellargs = dict(
+            parent=self,
+            id=wx.ID_ANY,
+            introText=self.intro,
+            locals={"gs": grass, "AddLayer": self.AddLayer},
+        )
+        # useStockId (available since wxPython 4.0.2) should be False on macOS
         if sys.platform == "darwin" and CheckWxVersion([4, 0, 2]):
-            self.shell = PyShell(parent=self, id=wx.ID_ANY,
-                                 introText=self.intro,
-                                 locals={'gs': grass,
-                                         'AddLayer': self.AddLayer},
-                                 useStockId=False)
-        else:
-            self.shell = PyShell(parent=self, id=wx.ID_ANY,
-                                 introText=self.intro,
-                                 locals={'gs': grass,
-                                         'AddLayer': self.AddLayer})
+            shellargs["useStockId"] = False
+        self.shell = PyShell(**shellargs)
 
         sys.displayhook = self._displayhook
 

--- a/gui/wxpython/lmgr/pyshell.py
+++ b/gui/wxpython/lmgr/pyshell.py
@@ -30,6 +30,7 @@ import grass.script as grass
 from grass.script.utils import try_remove
 
 from gui_core.wrap import Button, ClearButton
+from core.globalvar import CheckWxVersion
 
 
 class PyShellWindow(wx.Panel):
@@ -44,12 +45,17 @@ class PyShellWindow(wx.Panel):
         self.intro = _("Welcome to wxGUI Interactive Python Shell %s") % VERSION + "\n\n" + \
             _("Type %s for more GRASS scripting related information.") % "\"help(gs)\"" + "\n" + \
             _("Type %s to add raster or vector to the layer tree.") % "\"AddLayer()\"" + "\n\n"
-        # useStockId should be False on macOS
-        self.shell = PyShell(parent=self, id=wx.ID_ANY,
-                             introText=self.intro,
-                             locals={'gs': grass,
-                                     'AddLayer': self.AddLayer},
-                             useStockId=(sys.platform != "darwin"))
+        if sys.platform == "darwin" and CheckWxVersion([4, 0, 2]):
+            self.shell = PyShell(parent=self, id=wx.ID_ANY,
+                                 introText=self.intro,
+                                 locals={'gs': grass,
+                                         'AddLayer': self.AddLayer},
+                                 useStockId=False)
+        else:
+            self.shell = PyShell(parent=self, id=wx.ID_ANY,
+                                 introText=self.intro,
+                                 locals={'gs': grass,
+                                         'AddLayer': self.AddLayer})
 
         sys.displayhook = self._displayhook
 

